_slides/11-daveeggers.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+My name is Dave
```

Comment:
Not sure I like the changes here.  They are in black and white.  Really bland.  We need some color in here